### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Bornittah**
### Project requirements
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,4 +21,10 @@ def show
     @post = Post.find_by(author_id: params[:user_id], id: params[:id]) || 'No posts yet'
     @comments = @post.comments
   end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :text).merge(author_id: current_user.id)
+  end
```
Comment (anagudelogu) [inline app/controllers/posts_controller.rb]: - [ ] Great job using strong parameters! 🚀 However, I'm unable to create new posts, it looks like you have to initialize your counters for the post to be valid! 😱  Alternatively, please consider setting a default value of 0 in your database. Every time a post is created, both likes and comments counter start at 0 and you don't have to explicitly initialize them! 🤓 🔧

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,9 @@
   # root "articles#index"
   root to: "users#index"
   resources :users, only: [:index, :show]  do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
```
Comment (anagudelogu) [inline config/routes.rb]: - [ ] Please remember to create the routes to your Posts#new and Posts#create actions, at the moment those routes are inaccessible, ending up with the user not being able to create a Post. Kindly fix it! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @Bornittah, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Bornittah 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**